### PR TITLE
adding react 07-authorization guide

### PR DESCRIPTION
**Pull request by flaviogranero**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by flaviogranero** [comment]
@alemata could you review? sorry it's including 05 as well, but it's in another PR

File: articles/client-platforms/react/07-authorization.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+```javascript
+/* ===== ./src/utils/AuthService.js ===== */
+...
+export default class AuthService extends EventEmitter {
+  ...// omitting some methods
+  isAdmin(){
+    // Checks if user have `admin` role in his profile app_metadata
+    const profile = this.getProfile();
+    const { roles } = profile.app_metadata || {};
```
Comment (alemata) [inline articles/client-platforms/react/07-authorization.md:35]: is this ok? shouldn't be `profile.app_metadata.roles` ?

Comment (flaviogranero) [inline articles/client-platforms/react/07-authorization.md:35]: it was failing when profile.app_metadata is undefined.

**Comment by alemata** [comment]
👍 @mpaktiti @ntotten

File: articles/client-platforms/react/07-authorization.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+description: This tutorial will show you how assign roles to your users, and use those claims to authorize or deny a user to access certain routes in the app.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/07-Authorization',
+}) %>_
+
+<%= include('../_includes/_authorization-introduction', { ruleslink: '/quickstart/spa/react/06-rules' }) %>_
+
+## 2. Check if user's role is present
```
Comment (mpaktiti) [inline articles/client-platforms/react/07-authorization.md]: @flaviogranero There is no `1.` in the doc. I assume this should be it `07-authorization#create-a-rule-to-assign-roles`. Please update.

Also, the `#create-a-rule-to-assign-roles` is H3 (`### Create a Rule to assign roles`), while the rest in this doc are H2. This needs to be aligned.

File: articles/client-platforms/react/05-linking-accounts.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
+---
```
Comment (mpaktiti) [inline articles/client-platforms/react/05-linking-accounts.md]: @flaviogranero 
The document does not load: https://auth0-docs-content-pr-1780.herokuapp.com/docs/quickstart/spa/react/05-linking-accounts
I believe the reason is not escaping strings: https://github.com/auth0/docs#escaping-strings

For example, this:
`https://${this.domain}/api/v2/users/${userId}/${url}`

Should be:
`https://<%= "${this.domain}" %>/api/v2/users/<%= "${userId}" %>/<%= "${url}" %>`

File: articles/client-platforms/react/08-calling-apis.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
` file.
+
+<% if (account.userName) { %>
+If you don't have a `.env` file in your project you can create your own using the following command:
+
+```bash
+$ echo "AUTH0_CLIENT_ID='${account.clientId}\nAUTH0_DOMAIN='${account.namespace}'\nAUTH0_SECRET='account.secret'" > .env
+```
+<% } else { %>
+If you don't have a `.env` file in you project, please copy `.env.example` and add your own credentials from [auth0 dashboard](https://manage.auth0.com).
```
Comment (mpaktiti) [inline articles/client-platforms/react/08-calling-apis.md]: Please rename the link to `dashboard`, not `auth0 dashboard`. This is the one we use.

File: articles/client-platforms/_includes/_authorization-create-rule.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+First, we will create a rule that assigns our users either an `admin` role, or a single `user` role. To do so, go to the [new rule page](https://manage.auth0.com/#/rules/new) and select the "*Set Roles To A User*" template, under *Access Control*. Then, replace this line from the default script:
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_authorization-create-rule.md]: @flaviogranero 
Please update `[new rule page](https://manage.auth0.com/#/rules/new)` to `[new rule page](${uiURL}/#/rules/new)`

**Comment by mpaktiti** [comment]
The following PRs should be merged before this one:
- https://github.com/auth0/docs/pull/1721
- https://github.com/auth0/docs/pull/1767

**Comment by flaviogranero** [comment]
@mpaktiti @chenkie done.

File: articles/client-platforms/react/07-authorization.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
+---
+title: Authorization
+description: This tutorial will show you how assign roles to your users, and use those claims to authorize or deny a user to access certain routes in the app.
+---
+
```
Comment (mpaktiti) [inline articles/client-platforms/react/07-authorization.md:5]: @flaviogranero The System Requirements section is missing.

**Comment by flaviogranero** [comment]
@mpaktiti ready with system requirements

**Pull request merged**